query_optimizer/ExecutionGenerator.cpp

```diff
@@ -1527,5 +1527,11 @@ void ExecutionGenerator::convertTableGenerator(
   temporary_relation_info_vec_.emplace_back(tablegen_index, output_relation);
 }
 
+S::InsertDestination* ExecutionGenerator::addNewInsertDestinationToQueryContext() {
+  std::unique_ptr<S::InsertDestination> insert_destination_proto(query_context_proto_->add_insert_destinations());
```

Comment:
Yes, just remove the `unique_ptr`, and use the raw pointer. Because we do not manage the memory space for proto, although I don't know whether internally proto's `add_xxx` method uses a `new` operator.